triciaswift/tastevine-server
# Added list method to category view

I added a list method for Category View

Supported routes
`/categories` for a GET request will return an array of category objects

## Steps to test
1. Pull down the `ts-list-categories` branch and switch to it
2. If your debugger is running, restart it
3. Open Postman
4. Use this token in the header:
```
Token c29b168a0c1e461b3ff36dd9ce15e6e1d1768597
```
5. Perform a GET to `http://localhost:8000/categories` and make sure you receive a similar lookin response body:
```
[
    {
        "label": "Appetizers"
    },
    {
        "label": "Breakfast"
    },
    {
        "label": "Lunch"
    },
    {
        "label": "Sides"
    },
    {
        "label": "Dinner"
    },
    {
        "label": "Desserts"
    }
]
```
6. Make sure the status code is a 200
